src/Optimization/hiopAlgFilterIPM.cpp

```diff
@@ -1511,6 +1511,33 @@ hiopAlgFilterIPMNewton::switch_to_safer_KKT(hiopKKTLinSys* kkt_curr,
     } // end of if(kkt)
   }
 #endif
+
+// TODO: turn to 0 --- keep using fast mode till linear solver (kkt->update) fails
+#if 1
```

Comment:
@cnpetra both work fine with all the unit tests. Will test these two options with exago test cases. Please let me know if you have preference.

Reply:
> @cnpetra both work fine with all the unit tests. Will test these two options with exago test cases. Please let me know if you have preference.

looks good. I have a couple of questions/remarks for when we meet in person.